.github/workflows/documentation.yml

```diff
@@ -1,10 +1,9 @@
 name: Documentation
 on:
-  push:
-    paths:
-      - 'owl/**'
-      - 'docs/**'
-      - '.github/**'
+  workflow_run:
```

Comment:
I am not sure about this change. So basically before the trigger was *push* operation that changes one of the listed paths. If I understand correctly you changed it to "whenever the 'evaluation' workflow was triggered successfully" which is not right because the workflow is executed for pull requests which are not yet accepted! I think with this block the docu would update before the pull request is accepted

Reply:
I think it is ok to revert the changes back such that the docu is only generated on push again.

Reply:
Unfortunately, this is not a solution: The Documentation *must* run *after* the Evaluation, because the Evaluation downloads an artifact generated by the Evaluation. With your approach, however, both would run in parallel - there would be no triggering run, and we could not reference where to get the artifact from (see documentation.yaml line 25).
What we can do, however, is to ignore both jobs if the triggering run was itself not triggered by a push. I added this now via `if: ${{ github.event.workflow_run.event == 'push' }}`, see documentation.yaml line 10 and 47.